get_github_coderabbit_review/coderabbit_fetcher/ai/sentiment_analyzer.py

```
@@ -0,0 +1,704 @@
+                )
+
+            # Fallback to text parsing
+            return self._parse_text_response(response.content, text)
+
+        except Exception as e:
+            logger.error(f"Error parsing AI sentiment response: {e}")
+
+            # Return rule-based analysis as fallback
+            return self._analyze_with_rules(text)
+
+    def _parse_text_response(self, text: str, original_text: str) -> SentimentScore:
```

Ruff (0.13.2)

Unused method argument: `original_text`

(ARG002)